packages/appstore/src/components/fiat-transaction-list/fiat-transaction-list-item.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Statement } from '@deriv/api-types';
+import { isMobile } from '@deriv/shared';
```

Comment:
@rostislav-deriv use `{ is_mobile } = ui` https://deriv-group.slack.com/archives/C3AHMSM3P/p1683037206653169

Reply:
Done

Reply:
Rollbacking to `isMobile()` in this file due to this https://deriv-group.slack.com/archives/C04QJ7569B8/p1685675108289389?thread_ts=1685621855.432669&cid=C04QJ7569B8

Reply:
@rostislav-deriv But this is in appstore, Not components :))
Don't rollback :(